frontend/lib/widgets/app_bars.dart

```diff
@@ -101,10 +104,10 @@ class SearchSliverAppBarState extends State<SearchSliverAppBar> {
         focusNode: focusNode,
         decoration: InputDecoration.collapsed(
           hintText: t.search.searchHint,
-          hintStyle: TextStyle(color: foregroundColor?.withOpacity(0.8)),
+          hintStyle: theme.textTheme.bodyLarge,
```

Comment:
I think the opacity was appropriate here
![image](https://github.com/user-attachments/assets/0689e8fc-3665-49bf-b639-89cf3cd5d308)

Reply:
yes, I agree 🙈
(I haven't checked all the views yet, maybe something else will arise)

Reply:
:upside_down_face: I don't want to annoy with nit-picky comments, so please leave it as is if you want to, but I can't resist to at least mentioning that 😄
but I think it was better with transparency. because currently it's like a label, and not like a background hint.
![image](https://github.com/user-attachments/assets/32ee6a70-8d48-4b49-aba8-35aa0d623ed8)

Reply:
I decided not use opacity due to bad readibility but i can adjust that

Reply:
up to you.
I think it's standard for such text to be grayed out.
![image](https://github.com/user-attachments/assets/5ad93a77-f78a-4ad1-a961-feff7e096333)